src/ALE/coord_rho.F90

```diff
@@ -112,10 +116,25 @@ subroutine build_rho_column(CS, nz, depth, h, T, S, eqn_of_state, z_interface, &
   real, dimension(nz+1) :: xTmp   ! Temporary positions [H ~> m or kg m-2]
   real, dimension(CS%nk) :: h_new ! New thicknesses [H ~> m or kg m-2]
   real, dimension(CS%nk+1) :: x1  ! Interface heights [H ~> m or kg m-2]
+  real :: z0_top, eta ! Thicknesses or heights [Z ~> m] or [H ~> m or kg m-2]
+  logical :: new_rho_def
```

Comment:
The variable `new_rho_def` appears to be set but never used, so perhaps it should be deleted.  If it is retained, a more descriptive name might more appropriate. (Everything new becomes old.)